GVRf/Framework/jni/objects/components/render_data.h

```diff
@@ -85,13 +120,19 @@ class RenderData: public Component {
         rendering_order_ = rendering_order;
     }
 
-    bool cull_test() const {
-        return cull_test_;
-    }
+    bool cull_face(int pass = 0) const {
+    	if (pass >= 0 && pass < render_pass_list_.size()) {
+    		return render_pass_list_[pass]->cull_face();
+    	}
 
-    void set_cull_test(bool cull_test) {
-        cull_test_ = cull_test;
-    }
+    	return nullptr;
+	}
```

Comment:
Even though I think nullptr here equals with false, but I think "return false" is more accurate? Just my personal opinion.